lib/y2_installbase.pm

```diff
@@ -285,7 +285,7 @@ patterns.
 
 sub select_not_install_any_pattern {
     my ($self) = @_;
-
+    send_key_until_needlematch('minimal-system', 'down') if check_var_array('SCC_ADDONS', 'pcm');
```

Comment:
iterating the patterns is complicated process, there is a dedicated module for that, really old but sort of reliable, as old as the YaST Ui in that part, I would not take risk here iterating anything with something new, this code that you are updating had the goal to not iterate and rely on what was in the screen. The iteration should be visible as part of the test anyway, at least that was the idea how it was designed.
The other two options would be better:
(1) use select_patterns for all 3 arch. where this test suite is running including the pcm
(2) as it is a minimal scenario we don't need the pcm, remove it from setting and that would fix as well (I think it is more simple and Lemon use that solution already).